.github/workflows/facebook.yml

```diff
@@ -3,31 +3,52 @@ name: Build Facebook plug
 on:
   push:
     tags:
-      - 'facebook-v*.*.**'
+      - 'facebook-v**'
```

Comment:
How do we differentiate between a dev plug and a normal plug?

Reply:
as far as I understand the dev version is just for the DEV deployment, but the code is the same as for PRD. So, in that case, there is no need to tag it separately or even keep it on separate repositories. 

You can always build a Facebook version, but if the version is broken or doesn't work well, just don't deploy it to PRD. 
Moreover, to inform the community which version (git tag) is stable, we can simply create a release on a particular git tag